Source/Presentation/ITMO.Dev.ASAP.Controllers/IdentityController.cs

```diff
@@ -29,11 +29,11 @@ public async Task<ActionResult<LoginResponse>> LoginAsync([FromBody] LoginReques
         return Ok(loginResponse);
     }
 
-    [HttpPost("users/{username}/promote")]
-    [Authorize(Roles = AsapIdentityRole.AdminRoleName)]
-    public async Task<IActionResult> PromoteAsync(string username)
+    [HttpPost("users/{username}/change-role-to/{role}")]
```

Comment:
и значение новой роли через query параметр передавай 

```suggestion
    [HttpPut("users/{username}/role")]
```